code/index.html

```diff
@@ -1,30 +1,82 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!-- video header -->
+  <header>
+    <video id="video1" class="header-video" loop autoplay>
+      <source src="./videos/SampleVideo_1280x720_1mb.mp4" type="video/mp4">
+    </video>
+    <video id="video2" class="header-video" loop autoplay>
+      <source src="./videos/mediumVideo.mp4" type="video/mp4">
+    </video>
+    <div class="header-text">
+      <h1>Looking for a place to stay?</h1>
+      <h2>Try something different</h2>
+    </div>
+  </header>
+  <main>
     <!-- Signup form -->
+    <form id="signup-form" action="https://httpbin.org/anything" method="POST">
+      <div class="form-group">
+        <label for="name">Name</label>
+        <input id="name" type="text" name="Name">
+      </div>
+    <div class="form-group">
+        <label for="email">Email Address</label>
+        <input id="email" type="email" name="Email">
+      </div>
+      <div class="form-group">
+        <label for="password">Password</label>
+        <input id="password" type="password" name="Password">
```

Comment:
A bit of validation would be nice since now I can submit without any inputs. There's some reading here, check the attributes:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input